docs/api/cozy-client.md

```diff
@@ -833,13 +834,32 @@ Maximum number of documents returned, useful for pagination. Default is 100.
 ### queryDefinition.offset(skip) ⇒ [<code>QueryDefinition</code>](#QueryDefinition)
 Skip the first ‘n’ documents, where ‘n’ is the value specified.
 
+Beware, this [performs badly](http://docs.couchdb.org/en/stable/ddocs/views/pagination.html#paging-alternate-method) on view's index.
+ Prefer cursor-based pagination in such situation.
+
 **Kind**: instance method of [<code>QueryDefinition</code>](#QueryDefinition)  
 **Returns**: [<code>QueryDefinition</code>](#QueryDefinition) - The QueryDefinition object.  
 
 | Param | Type | Description |
 | --- | --- | --- |
 | skip | <code>number</code> | The number of documents to skip. |
 
+<a name="QueryDefinition+offsetCursor"></a>
+
+### queryDefinition.offsetCursor(cursor) ⇒ [<code>QueryDefinition</code>](#QueryDefinition)
+Use [cursor-based](https://docs.cozy.io/en/cozy-stack/jsonapi/#pagination) pagination.
+The cursor is a [startkey, startkey_docid] array, where startkey is the view's key,
+e.g. ["io.cozy.photos.albums", "album-id"] and startkey_docid is the id of
+the starting document of the query, e.g. "file-id".
+Use the last docid of each query as startkey_docid to paginate or leave blank for the first query.
```

Comment:
I do not understand how to find the view's key ?

Reply:
It's how is created the view at the begining, when you `emit()`

Reply:
> It's how is created the view at the begining, when you emit()

It means those cursors are specific to Cozy ?

Reply:
> It means those cursors are specific to Cozy ?

No. It means you hav to know how the view was built in the first place (ie: what is indexed and how.

If your view is built on `[fieldA, filedB, filedC]` then your cursor need to be `[fieldA_value, fieldB_value, fieldC, value], docid`